lib/vector-core/proto/event.proto

```diff
@@ -95,7 +107,7 @@ message Metric {
   }
   string namespace = 11;
   uint32 interval_ms = 18;
-  Value metadata = 19;
+  Metadata metadata = 19;
```

Comment:
I've added a breaking proto change as an example to test the workflow. I'll revert these changes before merging.